apps/rust_blinky/cargo_build.sh

```diff
@@ -32,5 +33,16 @@ else
   exit 1
 fi
 
-cargo build --target="${TARGET}" --target-dir="${MYNEWT_PKG_BIN_DIR}"
-cp "${MYNEWT_PKG_BIN_DIR}"/${TARGET}/debug/*.a "${MYNEWT_PKG_BIN_ARCHIVE}"
+# Map mynewt build profile to cargo build profile
+if [[ ${MYNEWT_BUILD_PROFILE} == 'debug' ]]; then
+  # mynewt debug profile -> cargo debug profile
+  CARGO_PROFILE="debug"
+  CARGO_ARGS=""
+else
+  # all other mynewt profiles -> cargo release profile
+  CARGO_PROFILE="release"
+  CARGO_ARGS=" --release"
+fi
+
+cargo build --target="${TARGET}" --target-dir="${MYNEWT_PKG_BIN_DIR}"${CARGO_ARGS}
```

Comment:
This would look better if there was a space between `${MYNEWT_PKG_BIN_DIR}"` and `${CARGO_ARGS}`, and of course, no space inside `CARGO_ARGS`, any reason for doing it this way?